content/en/blog/_posts/2016-11-00-Visualize-Kubelet-Performance-With-Node-Dashboard.md

```diff
@@ -5,6 +5,10 @@ slug: visualize-kubelet-performance-with-node-dashboard
 url: /blog/2016/11/Visualize-Kubelet-Performance-With-Node-Dashboard
 ---
 
+{{< caution >}}
+The Node Performance Dashboard was retired in early 2019 as part of the [kubernetes/contrib](https://github.com/kubernetes-retired/contrib/issues/3007) repository deprecation and is no longer available.
+{{< /caution >}}
```

Comment:
I'd avoid using a caution shortcode
- we might change how we use that in the future
- the caution shortcode is quite strong, implying “you could break your cluster if you don't pay attention here”
 
```suggestion
_Since this article was published, the Node Performance Dashboard was retired and is no longer available.
This retirement happened in early 2019, as part of the_ `kubernetes/contrib`
_[repository deprecation](https://github.com/kubernetes-retired/contrib/issues/3007)_.
```

Reply:
Assign @sftim since he already engaged on this/
/assign @sftim